app/views/events/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Events</h1>
+<br>
+
+<h1>Future Events</h1>
+
+<table style="width: 100%">
+  <thead>
+    <tr>
+      <th>Date</th>
+      <th>Title</th>
+      <th>Location</th>
+      <th>Description</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <%= render partial: 'event', collection: Event.upcoming, as: :event %>
+  </tbody>
+</table>
+
+<h1>Past events</h1>
+
+<table style="width: 100%">
+  <thead>
+    <tr>
+      <th>Date</th>
+      <th>Title</th>
+      <th>Location</th>
+      <th>Description</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <%= render partial: 'event', collection: Event.previous, as: :event %>
```

Comment:
all code related to accessing data from a database should be inside the model not in the view.